elements/pfe-dropdown/src/pfe-dropdown-item.scss

```
@@ -0,0 +1,69 @@
+@import "../../pfe-sass/pfe-sass";
+@import "_shared-assets";
+
+$LOCAL: dropdown-item;
```

@eyevana  I just noticed, the spacing variables were still different between the dropdown and the dropdown-item, but we could easily resolve by removing the $LOCAL var and nesting the other vars in an `item` region instead:

```suggestion
//$LOCAL: dropdown-item;

$dropdown-item-variables: (
  item: (
    FontSize: #{pfe-var(font-size)},
    FontWeight: 400,
    LineHeight: #{pfe-var(line-height)},
    Color: #151515,
    Color--hover: #151515,
    TextAlign: left,
    BackgroundColor: transparent,
    BackgroundColor--hover: #{pfe-color(surface--lighter)},
    Border: none
  )
);
```

---

You can test with this:
```
pfe-dropdown, pfe-dropdown-item {
    --pfe-dropdown--SpacingHorizontal: 40px;
}
```